projects/dashboards/ecometrics/src/app/app.component.html

```diff
@@ -0,0 +1,5 @@
+<h1 class="mat-h1">
+    <img src="assets/geoengine.svg" class="logo" />
+    <div>ECOMETRICS</div>
+</h1>
+<app-dashboard></app-dashboard>
```

Comment:
![grafik](https://github.com/user-attachments/assets/77a3e770-a7c5-4117-b3ba-4855ec7175a4)

No land use for 2023? Why is there a slider option?

Reply:
yeah we need more data. there is a task for this in trello